### DISCUSSION TITLE
please tell me about how to contribute this repository

### DISCUSSION THREAD

**Discussion by takaodaze** [SBoudrias/Inquirer.js#1120] [Q&A] [answered]
I am interested in this.
First, I opened the package and was baffled because it's hard to imagine what each package would look like.
And so, please tell me meaning each packages.

**Comment by SBoudrias** [comment, marked as answer]
Hey @takaodaze, thanks for asking 😄 

I've put together a first version of a [CONTRIBUTING](https://github.com/SBoudrias/Inquirer.js/blob/master/CONTRIBUTING.md) documentation. Let me know if there's more details that'd be useful to know to get started and we can improve that doc as we go!

Cheers
